containers/docker-compose.yaml

```
@@ -5,19 +5,23 @@ services:
     restart: always
     ports:
       - 5430:5432
-    volumes:
-      - db_data:/var/lib/postgresql/data
     environment:
       POSTGRES_PASSWORD: postgrespassword
+  # DRIVER={ODBC Driver 17 for SQL Server};SERVER=msserver;Uid=SA;Pwd=testPassword123
```

FYI, this looks like it was the cause of the `Login timeout expired` errors I was seeing when trying to add an MSSQL source. I was using the following as a connection string, rather than the service name (TIL):

```suggestion
  # DRIVER={ODBC Driver 17 for SQL Server};SERVER=localhost:1433;Uid=SA;Pwd=testPassword123
```

---

Docker container networking has been the cause of so much pain.
You have to remember the boundary between service-to-service calls, and host-to-service calls.

It gets really confusing sometimes, especially in scenarios like SSR apps where part of the render cycle can be run on the server, and part on the client, and in each case the URL to invoke another service is different (one using container networking, one using host networking)

😢 

This is partially why I'm not a huge fan of hybrid JS frameworks with SSR and partial rendering, it adds a lot of complexity and cognitive overhead and not all developers seem aware of the rendering patterns and different global scope/contexts.